api.md

```diff
@@ -919,6 +922,53 @@ The expected output:
 
 ---
 
+## interpolate() [](interpolate)
+
+The `interpolate` function does linear interpolation for missing values. It can only be used in an aggregation query with `time_bucket_gapfill`.
+The `interpolate` function call cannot be nested inside other function calls.
```

Comment:
I think we can add a link to the `time_bucket_gapfill` docs when we mention it here?